frontend/test/__support__/server-mocks/index.ts

```diff
@@ -15,6 +15,8 @@ export * from "./metabot";
 export * from "./metric";
 export * from "./model-indexes";
 export * from "./native-query-snippet";
+export * from "./revision";
+export * from "./public";
```

Comment:
👍 Nice catch on this missing export. What do you think about modifying the places that use these mocks? I think there are literally 3 other files that still import from `__support__/server-mocks/*` rather than `__support__/server-mocks/`. I'm just worried if we don't do it now we will forget it.

Reply:
Updated all imports of `server-mocks`